src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="get">
+
+      <fieldset class="groups">
```

Comment:
Blank line isn't needed here
```suggestion
    <form action="get">
      <fieldset class="groups">
```

Reply:
Change this in all places